lib/skate/application.ex

```diff
@@ -38,6 +38,7 @@ defmodule Skate.Application do
         end ++
         [
           {Phoenix.PubSub, name: Skate.PubSub},
+          {DNSCluster, Application.get_env(:skate, DNSCluster) || [query: :ignore]},
```

Comment:
suggestion: rather than the extra query here, I'd throw this into `config/config.exs`:
```elixir
config :skate, DNSCluster, query: :ignore
```

Reply:
I'd thought of that but was on the fence, I think that's the right call though, thanks!

Reply:
Done! [`057f2eb` (#2831)](https://github.com/mbta/skate/pull/2831/commits/057f2eb598097af9aa5642d8de7754eb03cf783a)